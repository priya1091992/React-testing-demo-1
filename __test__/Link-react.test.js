import React from 'react';
import Link from '../src/Link.react';
import renderer from 'react-test-renderer';

test('Link changes the class when hovered', () => {
  const wrapper = renderer.create(
    <Link page="http://www.gmail.com">Facebook</Link>,
  );
  let tree = wrapper.toJSON();

  expect(tree).toMatchSnapshot();
});
