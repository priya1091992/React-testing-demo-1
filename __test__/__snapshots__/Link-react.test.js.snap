// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
<a
  href="http://www.gmail.com"
>
  http://www.gmail.com
</a>
`;
